src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -46,28 +43,26 @@ object Features {
 
       val minimumImpressionsThreshold = platform match {
         case Android => 10000
-        case iOS => 40000
+        case Ios => 40000
       }
 
-      MonitoringQuery(query, minimumImpressionsThreshold)
+      val checks: List[Check] = List(TotalImpressionsIsGreaterThan(minimumImpressionsThreshold))
```

Comment:
If we wanted to check something different using the same query results (e.g. that we served an appropriate number of friction screens on recent beta versions) we'd add that check to this list.